presto-docs/src/main/sphinx/plugin/redis-hbo-provider.rst

```diff
@@ -0,0 +1,61 @@
+=======================
+Redis Hbo Provider
+=======================
+
+Redis Hbo Provider allows loading a custom configured Redis Client for storing/retrieving Historical Stats. The Redis client is stateful and is based on
+Lettuce https://github.com/lettuce-io/lettuce-core
+
+Configuration
+-------------
+
+Create ``etc/catalog/hbo-provider.properties`` to mount the Redis Hbo Provider Plugin, replacing the properties as appropriate:
+
+Configuration properties
+------------------------
+
+The following configuration properties are available:
+
+===================================     =====================================================================
+Property Name                           Description
+===================================     =====================================================================
+``coordinator``                         Boolean property whether presto server a coordinator
+``hbo.provider.server_uri``             Redis Server URI
+``hbo.provider.total-fetch-timeoutms``  Maximum timeout in ms for redis fetch requests
+``hbo.provider.total-set-timeoutms``    Maximum timeout in ms for redis set requests
+``hbo.provider.default-ttl-sec``        TTL in sec of the redis data to be stored
```

Comment:
default-ttl-sec -> default-ttl-seconds
TTL in sec of ... -> TTL in seconds of ...